.buildconfig/ci-linux.yml

```diff
@@ -7,7 +7,7 @@ channels:
 dependencies:
   - cmake=4.0.0
   - conan=1.66.0
-  - cppcheck=2.6.2
+  - cppcheck=2.18.0
```

Comment:
Was this necessary for updating to py 3.11?

Reply:
yes, the min cppcheck that is built with py3.11 is `2.18.0` :/